Podfile.lock

```diff
@@ -181,4 +181,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: b7d70eb804b087e359ae71cece52dfdbdb17f69c
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.8.3
```

Comment:
am i misunderstanding something, but why would this number ever decrease?

Reply:
Cuz im a bum. WIll update my cocoapods sir!